pkg/runner/local_docker.go

```diff
@@ -118,7 +118,7 @@ func (r *LocalDockerRunner) Healthcheck(ctx context.Context, engine api.Engine,
 	sidecarContainerOpts := docker.EnsureContainerOpts{
 		ContainerName: "testground-sidecar",
 		ContainerConfig: &container.Config{
-			Image:      "ipfs/testground:latest",
```

Comment:
We should probably fix this. If someone builds a release, we want to be picking up the given release docker image (for example `iptestground/sidecar:v0.4.0`), rather than `edge` or `latest`.

So this should be a var that we overwrite during release.